client/src/components/Atoms/Wrapper.tsx

```diff
@@ -23,4 +23,13 @@ export const Inner = styled.div<IInnerProps>`
   flex-direction: ${props => props.flexDirection || undefined};
 `
 
+export const SupportToolsInner = styled.div`
+  display: flex;
+  justify-content: center;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 30px 0;
+  width: 80%;
```

Comment:
Should we just go full-width (with some padding)? I feel like using 80% width will make things very dependent on the window width, which could lead to unexpected results on different screens.